src/app/@ansyn/ansyn/modules/overlays/services/overlays.service.ts

```diff
@@ -149,4 +154,22 @@ export class OverlaysService {
 	getThumbnailName(overlay): string {
 		return this._overlaySourceProvider.getThumbnailName(overlay)
 	}
+
+	getSensorGroupAndProviderFromSensorName(sensorName: string): IDataInputFilterValue {
```

Comment:
getSensorType is not a better name?

Reply:
Yes, I think you're right. This came from  the mess around the terms sensor name / sensor type.